scripts/setup-circleci.sh

```diff
@@ -33,7 +33,9 @@ dnf_install ninja-build ccache gcc-toolset-9 git wget which libevent-devel \
 dnf remove -y gflags
 
 # Required for Thrift
-dnf_install autoconf automake libtool bison flex
+dnf_install autoconf automake libtool bison flex python3
+# install sphinx for doc gen
+pip3 install sphinx sphinx-tabs breathe sphinx_rtd_theme
```

Comment:
I'd prefer you use a separate image: smaller, single-purpose images are easier to maintain.